.github/workflows/test.yml

```diff
@@ -163,7 +163,11 @@ jobs:
 
       - name: Destroy AWS stacks used for testing
         run: |
-          poetry run cdk destroy --force --all
+          cdk --app cdk.out ls | while read line ; do
+             if [ "$line" != "" ] ; then
+                aws cloudformation delete-stack --stack-name $line
```

Comment:
`$line` should be quoted to avoid any weirdness when substituting it, as in `"$line"`.